r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -227,6 +228,19 @@ test_that("Group by n_distinct() on dataset", {
       collect(),
     tbl
   )
+  # Without groupby
+  expect_dplyr_equal(
+    input %>%
+      summarize(distinct = n_distinct(int, na.rm = FALSE)) %>%
```

Comment:
If `count_distinct` works on Boolean input now, let's change these tests back to using `lgl` to match the other tests directly above.
```suggestion
      summarize(distinct = n_distinct(lgl, na.rm = FALSE)) %>%
```

Reply:
done! thanks!